crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -409,6 +412,26 @@ impl MeshPipelineKey {
             _ => PrimitiveTopology::default(),
         }
     }
+
+    pub fn from_cull_mode(cull_mode: Option<Face>) -> Self {
+        let cull_mode = match cull_mode {
+            Some(Face::Back) => 0,
+            Some(Face::Front) => 1,
+            None => 2,
+        };
+        let cull_mode_bits = (cull_mode & Self::CULL_MODE_MASK_BITS) << Self::CULL_MODE_SHIFT_BITS;
+        MeshPipelineKey::from_bits(cull_mode_bits).unwrap()
+    }
+
+    pub fn cull_mode(&self) -> Option<Face> {
+        let cull_mode_bits = (self.bits >> Self::CULL_MODE_SHIFT_BITS) & Self::CULL_MODE_MASK_BITS;
+        match cull_mode_bits {
+            0 => Some(Face::Back),
+            1 => Some(Face::Front),
+            2 => None,
```

Comment:
Similar to above, reordering:
```suggestion
            0 => None,
            1 => Some(Face::Front),
            2 => Some(Face::Back),
```